lib/src/interface/common.dart

```diff
@@ -71,19 +73,20 @@ String? getExecutablePath(
   String pathSeparator = platform.isWindows ? ';' : ':';
 
   List<String> extensions = <String>[];
-  if (platform.isWindows && context.extension(command).isEmpty) {
+  if (platform.isWindows && context.extension(executable).isEmpty) {
     extensions = platform.environment['PATHEXT']!.split(pathSeparator);
   }
 
   List<String> candidates = <String>[];
-  if (command.contains(context.separator)) {
-    candidates = _getCandidatePaths(
-        command, <String>[workingDirectory], extensions, context);
+  late List<String> searchPath;
```

Comment:
I don't think these needs to be late, since it appears to be definitely assigned in the if/else below

Reply:
Huh. Nice that it'll allow that. I thought `late` had to be specified when a non-nullable was not initialized. But maybe that's just class members.